src/libCli/libCli/ConnectionManager.hpp

```diff
@@ -52,13 +52,11 @@ namespace cli
         /// @returns the gRpc DescriptorPool of the corresponding server address.
         std::shared_ptr<grpc::protobuf::DescriptorPool> getDescPool(std::string f_serverAddress, ArgParse::ParsedElement &f_parseTree);
 
-        /// @brief closes the DescDb stream with a given deadline.
+        /// @brief closes the DescDb stream with a default deadline.
```

Comment:
What is the default deadline? is it the one set when initializing? maybe add details here

Reply:
20secs, adding comment